src/service/projectService.ts

```diff
@@ -0,0 +1,60 @@
+import CardWebModel from "../model/cardWebModel";
+import JqlService from "./jqlService";
+import JqlModel from "../model/jqlModel";
+
+import EpicJqlToCardWebModel from "../converter/epicJqlToCardWebModel";
+import ProjectToCardWebModel from "../converter/projectToCardWebModel";
+/**
+ * Created by JJax on 19.11.2016.
+ */
+
+export default class ProjectService {
```

Comment:
tutaj httpClient jest zależny od instancji jiry, tzn wie do której jiry ma wysyłać requesty, może być ciężko to zrobić z DI. Niemniej jest sporo innych miejsc gdzie można użyć DI, ale chyba nie za pomocą @Injectable.